.github/ISSUE_TEMPLATE/performance-question.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F947 Performance Question"
+about: Performance question about TiDB which is not caused by bug.
+
+---
+
+## Performance Question
+
+- What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
+
+- What's the difference between the performance you expect and the performance you observe?
+
+- Are you comparing TiDB with some other database? What's the difference between TiDB and compared one?
+
+- If it's a specific SQL query, could you please
+    - let us know whether you get your result after analyzing the table involved in the query.
```

Comment:
Should we dump the stats before analyze, to save more information?

Reply:
Oh, here i mean whether he analyzed or not before finding the performance issue.
Is this sentence ambiguous?

Reply:
I think so...

Reply:
changed